src/main/java/nl/novi/techiteasy/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,77 @@
+package nl.novi.techiteasy.controllers;
+
+import jakarta.validation.Valid;
+import nl.novi.techiteasy.dtos.TelevisionCreateDTO;
+import nl.novi.techiteasy.dtos.TelevisionResponseDTO;
+import nl.novi.techiteasy.exceptions.RecordNotFoundException;
+import nl.novi.techiteasy.exceptions.TelevisionNameTooLongException;
+import nl.novi.techiteasy.mappers.TelevisionMapper;
+import nl.novi.techiteasy.models.Television;
+import nl.novi.techiteasy.services.TelevisionService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/televisions")
+public class TelevisionsController {
+
+    private final TelevisionService televisionService;
+
+    public TelevisionsController(TelevisionService televisionService) {
+        this.televisionService = televisionService;
+    }
+
+    @PostMapping
+    public ResponseEntity<?> createTelevision(@Valid @RequestBody TelevisionCreateDTO televisionCreateDTO, BindingResult result) {
```

Comment:
het gebruik van een ? is niet heel netjes

Reply:
Thanks ik ga Object gebruiken.